cmdtest.sh

```
@@ -0,0 +1,8 @@
+ #!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] Remove leading spaces before the shebang.

(SC1114)